src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -31,6 +31,10 @@
     private Date reportEndDate;
     private String policy;
     private String expenseType;
+    private KualiDecimal cashAdvanceAmount;
+    private Date cashAdvanceIssueDate;
```

Comment:
Are cashAdvanceAmount and cashAdvanceIssueDate actually needed on this POJO? I don't see them actively used by non-logging code, and I don't think I need them for the Collector processing.